extensions/amp-analytics/amp-analytics.md

```diff
@@ -775,7 +779,34 @@ To configure a timer which times user events use:
 
 See the spec on [triggers](#triggers) for details on creating nested timer
 triggers. Note that using a timer trigger to start or stop a timer is not
-allowed.
+allowed. Please refer to the examples below on how to configure triggers based on a documents `hidden` and `visible` events.
```

Comment:
we have another example on timer trigger. Do you think we can put them together, and add descriptions to both examples. Thanks

Reply:
Which example are you referring to? We have a basic example and also a amp-video example.

Reply:
I was referring to the amp-video example